File: code.leetcode/shuzu/e26删除有序数组中的重复项.go
package shuzu
func removeDuplicates(nums []int) int {
	//设map
	m:=make(map[int]bool)
	//遍历数组
	left := 0
	for _, v := range nums { // v 即 nums[right]
		if !m[v] {
			nums[left] = v
			left++
			m[v]=true
		}
	}
	
	return left
	
	}

File: code.leetcode/shuzu/easy/e26删除有序数组中的重复项.py
class Solution(object):
    def removeDuplicates(self, nums):
        #设map
        m = {}
        #遍历数组
        left = 0
        for  v in nums:  # v 即 nums[right]
            if v not in m:
                nums[left] = v
                left+=1
                m[v]=True
            
        
        
        return left
        
	